src/internal/DotStarEsp32DmaSpiMethod.h

```diff
@@ -0,0 +1,320 @@
+/*-------------------------------------------------------------------------
+NeoPixel library helper functions for DotStars using Esp32, DMA and SPI (APA102).
+
+Written by Michael C. Miller.
+
+I invest time and resources providing this open source code,
+please support me by dontating (see https://github.com/Makuna/NeoPixelBus)
+
+-------------------------------------------------------------------------
+This file is part of the Makuna/NeoPixelBus library.
+
+NeoPixelBus is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
+
+NeoPixelBus is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with NeoPixel.  If not, see
+<http://www.gnu.org/licenses/>.
+-------------------------------------------------------------------------*/
+
+#pragma once
+
+#include "driver/spi_master.h"
+
+class Esp32VspiBus
+{
+public:
+    static const spi_host_device_t spiHostDevice = VSPI_HOST;
```

Comment:
public static const members always start with caps, so this "spiHostDevice" should be "SpiHostDevice".
This shows up multiple times, but I will only comment on it once here.